shipit.dontdeployme.yml

```diff
@@ -0,0 +1,17 @@
+dependencies:
+  override: []
+deploy:
+  override:
+    - echo 'Not to be deployed'
```

Comment:
Looks good. Let's just copy what's in the default shipit.yml and rename this to `shipit.prod.yml`